src/opencost_parquet_exporter.py

```diff
@@ -85,12 +98,20 @@ def get_config(
             datetime.now() - timedelta(1), '%Y-%m-%d')
         window_start = yesterday+'T00:00:00Z'
         window_end = yesterday+'T23:59:59Z'
+    if parquet_prefix is None:
+      window = datetime.strptime(window_start, "%Y-%m-%dT%H:%M:%SZ")
```

Comment:
is this correct ? , because it would be overwritten by line 109. 
Would be better to rename it as window_start_date or something else, just to be clear this is not the window that is passed to the api, but it is the window_start in a python date object.

Reply:
Yes it's ok. I'm using it only to build the default parquet prefix